cogs/helpers/_taskValidator.py

```diff
@@ -45,19 +45,19 @@ def validate(fileName):
                 args = task[2]
                 preposition = task[3]
                 end = task[4].split()
-                out += f"Execute command `{command} {args}` at {start[1] if start[1] != '*' else 'any'}:{start[0] if start[0] != '*' else 'any'}"
+                out += f"Execute command `{command} {args}` at {start[1] if start[1] != '*' else 'any month'}:{start[0] if start[0] != '*' else 'any day'}"
```

Comment:
Left padding with 0 should go on this line, only if the time is single digit. The issue is that if the time is 15:05 then it will currently print as 15:5. Likely a calculation in a line above to see whether it is a single or two digit number, and add a 0 at the start if necessary. Also the "any" here is referring to time not months or days, the month and day is handled in the line below so this can stay as is :) 
```suggestion
                out += f"Execute command `{command} {args}` at {start[1] if start[1] != '*' else 'any'}:{start[0] if start[0] != '*' else 'any day'}"
```